_pages/packaging-resources.md

```diff
@@ -0,0 +1,112 @@
+title: "Python packaging resources for scientists"
+excerpt: "We make the scientific Python package ecosystem easier to navigate through peer review and packaging resources."
+header:
+  overlay_image: images/header.jpg
+  overlay_filter: 0.8
+intro:
+  - excerpt: "There are many tools and approaches that can be used to create a Python package. We build resources that help you both understand the tool ecosystem and also learn how to create a Python package using modern best practices."
+peer-review:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people from different backgrounds sitting around it, working on laptops and also writing together."
+    excerpt: "We run an open peer review process for scientific Python software. Peer review helps maintainers improve the quality, usability and long-term maintainability of scientific software."
+    title: Raising the bar for scientific Python software"
+    url: https://www.pyopensci.org/about-peer-review/index.html
+    btn_label: Learn more and get involved with peer review today.
+    btn_class: btn--primary
+packaging-easier:
+   - image_path: images/pyopensci-working-together.png
+     alt: "A black and grey sketch of a group of people sitting at a desk in front of a monitor smiling."
+     title: "Get Involved: Help us improve our scientific Python packaging resources"
+     excerpt: "
+     We need your help! Our packaging content is community-created and reviewed through an open review process on GitHub. The more feedback that we get, the more useful our resources are to the community. Get involved by:
+
+     * Opening an issue about problems that you find in our guidebook
+
+     * Submitting a pull request that fixes a typo or mistake in the guide.
+
+     * Get credit for your contribution
+
+
+     All contributions are recognized both on our website and in the guidebook's citation.
+     "
+     url: https://github.com/pyOpenSci/python-package-guide/pulls
+     btn_label: See what we're working on now
+     btn_class: btn--primary
+community-created:
+   - image_path: images/pyopensci-people-working-together.png
+     alt: "A group of people working on a tetris like set of building blocks, trying to stack the blocks together. "
+     title: "Community created Python Packaging Guide"
+     excerpt: "
+     Our packaging guide is a living document that will help you navigate the Python packaging ecosystem and also learn about modern Python packaging best practices.
+
+    * Created & curated by the community
+
+    * Reviewed by beginner to expert level Pythonistas
+
+    * Accurate, modern & beginner-friendly
+
+
+     All contributions are recognized both on our website and in the guidebook's citation.
+     "
+toc: false
+---
+
+{% include feature_row id="intro" type="center" %}
+
+<div class="pyos-section purple">
+<div class="content" markdown="1">
+
+{% include feature_row id="community-created" type="right" %}
+
```

Comment:
the spacing between 'community created...' and 'guidebook sections' feels too large. is there a way to reduce the amount of negative space here?
<img width="1391" alt="Screenshot 2024-01-25 at 1 08 38 PM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/23085445/bb7cc627-8217-470e-bd10-094ef45770b3">

Reply:
putting in the ribbons (like those used as section breaks further down) might be a nice touch here - and in the other section breaks on the page. the ribbons give a nice visual flow, and gently break up each section, making it feel less abrupt.

Reply:
oops my comment below was for the specific point! this is a great point. The spacing here is a product of some of the sitewide styles (specifically the top margin on that h3 tag that is the guidebook sections title. i want to be careful about touching those because we'd want to do a sitewide check. We could do a one-off style but ideally the styles don't get too crazy. 

so how about for now we leave this as is BUT note that in the future it might be nice to create some more custom styles for just this issue. i totally agree is it too much of a gap so it's not clear where it belongs.  it does belong in this section but perhaps closer to the descriptive text above.

Reply:
that makes sense! another approach (I don't know how disruptive this would be) might be to adjust the font size in a couple of places:
- the text in the community-created section
- the guidebook sections header

shrinking the image size might also help lessen the gap.

all of this can be kind of fiddly so also OK to leave! just sharing ideas :)

Reply:
that worked. i've actually wanted to add a template item to our includes that are used for those image / text blocks to adjust image width. so i went ahead and did that and it won't be at all disruptive because you can chose to add it or not! (i added a conditional!).

Reply:
The other thing that is bothering me is the eggshell color doesn't jive with the purple background. but i think maybe we can go back at some point in the future and adjust all of our color palette colors because the website is colors i just picked forever ago. 

<img width="1295" alt="Screenshot 2024-01-25 at 4 16 43 PM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/f0f7a5fa-98d7-4da2-ac17-5ecef4d58806">

Reply:
actually looking at it - if that section was more narrow / had more padding on either side, it would visually sit under that "community created packaging guide" h2 title. i'll maybe try that too?